libsoftwarecontainer/include/softwarecontainer.h

```diff
@@ -119,21 +117,22 @@ class SoftwareContainer :
     ReturnCode activateGateways();
     ReturnCode shutdownGateways();
 
-    std::shared_ptr<Workspace> m_workspace;
-    ContainerID m_containerID;
-    std::string m_bridgeIp;
-    int m_netmaskBits;
-
-    ObservableWritableProperty<ContainerState> m_containerState;
+    // The new and improved Workspace
```

Comment:
This doesn't provide much for the understanding of the code as it stands now. Developers familiar with the now removed Workspace class will understand the reference, but others will be left with a mystery they will have to quest around in the git history for.